src/cluster_linearize.h

```diff
@@ -5,6 +5,7 @@
 #ifndef BITCOIN_CLUSTER_LINEARIZE_H
```

Comment:
nit: if you touch up this commit, I think the commit message has a couple typos:

`This is a class that encapsulated precomputes ancestor set feerates, and`

should read "encapsulates precomputed ancestor set feerates"